examples/rest-angular/src/styles.css

```diff
@@ -0,0 +1 @@
+/* You can add global styles to this file, and also import other style files */
```

Comment:
This file looks like an optional CSS override. Can we remove it from the example?